docs/source/releases/latest/1164-output-formatter-families-pass-empty-xarray-dataset-in-single-source.yaml

```diff
@@ -0,0 +1,19 @@
+bug fix:
+- title: "Don't pass empty xarray in single_source"
+  description: |
+    Output Formatter families `xrdict_area_product_to_outlist` and
+    `xrdict_area_product_outfnames_to_outlist` pass the variable
```

Comment:
For now, please add xrdict and outfnames to cspell.json.

@biosafetylvl5 found that the reason this isn't getting flagged elsewhere is that we're not currently blocking when checking spelling for modified files, only for new files. We'll need to put together a plan to get to a point where we can enforce spelling for real, but we're not there yet.